src/components/Sidebar/SidebarAuth/index.tsx

```diff
@@ -377,10 +381,9 @@ export const SidebarAuth = ({
     case 'confirm': {
       form = (
         <Paragraph>
-          Überprüfe Dein E-Mail Postfach für <Quotes>{formData.email}</Quotes>{' '}
-          nach einer E-Mail von{' '}
-          <Quotes>{process.env.NEXT_PUBLIC_FROM_EMAIL}</Quotes> mit einem Link
-          um deinen Account zu bestätigen.
+          {checkSignupMail
+            .replace('_1_', formData.email)
```

Comment:
to 1. I agree, will change

to 2. I am not sure if it as straightforward as you think using `ListFormat` due to the fact that the text is not rendered as a plain string, but has `<ValidOrNot>` tags in between:
```
   <SmallParagraph>
        {part1} <ValidOrNot success={errors.length} />, {part2}{' '}
        <ValidOrNot success={errors.lowerCase && errors.upperCase} />,{part3}{' '}
        <code>{specialCharacters}</code>{' '}
        <ValidOrNot success={errors.specialChar} /> {part4}{' '}
        <ValidOrNot success={errors.digit} />
      </SmallParagraph>
```

to 3. There are thousand of ways this could be implemented. The way it is implemented now (condensed, minimal example below) works fine and is concise and extendable enough in my eyes. But if it's important to you, I can change - please provide more detailed information on how you would implemented it and why it's cleaner.

```
interface Content {
  title: string
}

export enum Language {
  de = 'de',
  en = 'en',
}

export interface LocalizedContent {
  de: Content;
  en: Content;
}

const localizedContent = {
  de: {title: "Hallo Welt"},
  en: {title: "Hello world"},
} as LocalizedContent;
```

Reply:
to 1. 👍🏾 
to 2. Okay I get it. I guess you are right. Wrapping this up into to much logic does no good. Maybe give the variables then more meaningful names than part1,2,3 ?
to 3. Sorry I did not look at the types. This is actually what I meant. I got thrown of by the `deContent` variable naming. The lookup for content will be `content["de"]["title"]` here.

Reply:
to 1. Changed to functions instead of dirty text replace in https://github.com/technologiestiftung/giessdenkiez-de/pull/694/commits/705d156ccbc843c58ebf103382ee6f36d445b593 ✅

Reply:
to 2.

> Maybe give the variables then more meaningful names

yes, done in latest commits